jans-config-api/plugins/admin-ui-plugin/src/main/java/io/jans/ca/plugin/adminui/service/license/LicenseDetailsService.java

```diff
@@ -508,4 +513,16 @@ private io.jans.as.client.TokenResponse generateToken(String opHost, String clie
             return null;
         }
     }
+
+    private Optional<GenericResponse> handleLicenseApiNotAccessible(Response response) {
+        if (response.getStatus() == 404) {
+            log.error("{}", LICENSE_APIS_404);
```

Comment:
@duttarnab you don't need to involve formatter, it's as simple as `log.error(LICENSE_APIS_404);`